.golangci.reference.yml

```diff
@@ -1505,6 +1523,14 @@ linters-settings:
     # Default: 0.8
     confidence: 0.1
 
+    error-code: 1
```

Comment:
some of the options I was not able to get help/documentation from where I found them in the code